src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -159,7 +159,19 @@ public void addParticipant(Participant participant) throws AlfredException {
      * @return boolean
      */
     public boolean updateParticipant(Id id, Participant participant) {
-        return this.participantList.update(id, participant);
+        try {
+            // Update the participant in the team list as well
+            Team targetTeam = this.getTeamByParticipantId(id);
```

Comment:
What if the participant doesn't have a team?

Reply:
Then an error will be thrown, so this method will return false

Reply:
So if a participant is not in a team, we cant update his/her details?

Reply:
Fixed in another PR